src/main/java/by/dragonsurvivalteam/dragonsurvival/server/handlers/PlayerLoginHandler.java

```diff
@@ -39,7 +39,9 @@ public static void syncCompleteSingle(Entity entity) {
             DragonStateProvider.getOptional(player).ifPresent(handler -> {
                 if (handler.getType() != null && handler.getBody() == null) {
                     // Otherwise players won't be able to join the world
-                    handler.setBody(DragonBodies.CENTER);
+                    // TODO :: not the best idea - any default body could be removed by datapacks
+                    // TODO :: maybe better to revert the player to non-dragon
+                    handler.setBody(entity.registryAccess().holderOrThrow(DragonBody.center));
```

Comment:
revertToHumanForm?

Reply:
Could do that or assign a random body
(That is what I replaced the other default usages with)